ietf/templates/nomcom/qualified_volunteer_list_for_announcement.txt

```diff
@@ -0,0 +1,4 @@
+{% load ietf_filters %}{% autoescape off %}Name                     Affiliation
+
+{% for volunteer in volunteers %}{{ volunteer.person.plain_name|ljust:"25" }}{{ volunteer.affiliation }}
```

Comment:
I wonder if you can explicitly put "none supplied" or something if affiliation is empty?